pkg/cloud/services/elb/loadbalancer.go

```diff
@@ -454,6 +458,57 @@ func (s *Service) describeClassicELB(name string) (*infrav1.ClassicELB, error) {
 	return fromSDKTypeToClassicELB(out.LoadBalancerDescriptions[0], outAtt.LoadBalancerAttributes), nil
 }
 
+func (s *Service) reconcileELBTags(name string, desiredTags map[string]string) error {
+	tags, err := s.scope.ELB.DescribeTags(&elb.DescribeTagsInput{
+		LoadBalancerNames: []*string{aws.String(name)},
+	})
+	if err != nil {
+		return err
+	}
+
+	if len(tags.TagDescriptions) == 0 {
+		return errors.Errorf("no tag information returned for load balancer %q", name)
+	}
+
+	currentTags := converters.ELBTagsToMap(tags.TagDescriptions[0].Tags)
+
+	addTagsInput := &elb.AddTagsInput{
+		LoadBalancerNames: []*string{aws.String(name)},
+	}
+
+	removeTagsInput := &elb.RemoveTagsInput{
+		LoadBalancerNames: []*string{aws.String(name)},
+	}
+
+	for k, v := range desiredTags {
+		if val, ok := currentTags[k]; !ok || val != v {
+			s.scope.V(4).Info("adding tag to load balancer", "key", k, "value", v)
```

Comment:
Can we add the elb ID as a pair here too? Sames goes for L492.

Reply:
done